### PR TITLE
feat(wizards/services): add read-only wizard on access point and ied

### PR DESCRIPTION
**Pull request by pascalwilbrink**
Signed-off-by: Pascal Wilbrink <pascal.wilbrink@alliander.com>

### CODE REVIEW

**Comment by Sander3003** [comment]
@pascalwilbrink can you finish this pull-request? I would like get in into production.

File: test/integration/wizards/services-wizard.test.ts [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    it('Services wizard to have 4 pages', () => {
+      expect(element.wizardUI.wizard.length).to.equal(4);
+    });
+    it('Services wizard to have 85 inputs', () => {
+      expect(element.wizardUI.wizard.flatMap(p => p.content).length).to.equal(
+        85
+      );
+    });
+
+    [13, 22, 22, 28].forEach((inputs, idx) => {
```
Comment (JakobVogelsang) [inline test/integration/wizards/services-wizard.test.ts:35]: Nice idea!!

File: test/integration/wizards/services-wizard.test.ts [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+      it(`Services wizard ${idx + 1} to have ${inputs} inputs`, () => {
+        expect(element.wizardUI.wizard[idx].content!.length).to.equal(inputs);
+      });
+    });
+
+    [0, 1, 2, 3, 4].forEach(idx => {
```
Comment (JakobVogelsang) [inline test/integration/wizards/services-wizard.test.ts:41]: I fear those snapshots are not enough. They cover not all inputs. The test file I gave you has per AccessPoint inputs activated. You are using "only" the one in the IED element.

File: test/unit/editors/ied/__snapshots__/access-point-container.test.snap.js [hunk 3]
```diff
@@ -6,16 +6,32 @@ snapshots[
 ] = `<action-pane tabindex="0">
   <mwc-icon slot="icon">
   </mwc-icon>
+  <abbr
```
Comment (JakobVogelsang) [inline test/unit/editors/ied/__snapshots__/access-point-container.test.snap.js:9]: Please reverse this.

**Review by JakobVogelsang — CHANGES_REQUESTED** [review]

File: test/integration/wizards/services-wizard.test.ts (hunk 2 above)
Comment (JakobVogelsang) [reply test/integration/wizards/services-wizard.test.ts:41]: At the moment, you are coveting 4 out of 6 pages with the snapshot by choosing the `Services` element in the `IED`. By adding additional tests, you can make the rest of the pages visible. @pascalwilbrink

Comment (pascalwilbrink) [reply test/integration/wizards/services-wizard.test.ts:41]: Fixed

**Review by JakobVogelsang — APPROVED** [review]
LGTM thx

**Pull request merged**